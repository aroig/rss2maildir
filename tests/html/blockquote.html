<h1>A block quote test</h1>
<p>Earlier someone said:</p>
<blockquote>
The world isn't flat after all, it's actually round, so there.
</blockquote>
