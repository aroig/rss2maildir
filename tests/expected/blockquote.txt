A block quote test
==================

Earlier someone said:

    The world isn't flat after all, it's actually round, so there.
